app/models/notification_type.rb

```diff
@@ -32,7 +32,7 @@ def subscriber_ids(subject, initiator)
       User.superadmins.pluck(:id)
     when AUDIENCE_NONE
       []
-    end
+    end || []
```

Comment:
I don't think this is enough.  Under the `AUDIENCE_TENANT` both cases will return an instance rather than an array.

Reply:
No ... the tenant case has a `.try(:user_ids).try(:uniq)` after the conditional block.

Reply:
Oh, sorry I missed that